Meta/gn/secondary/Meta/Lagom/Contrib/MacPDF/BUILD.gn

```diff
@@ -0,0 +1,104 @@
+import(
+    "//Meta/gn/secondary/Meta/Lagom/Contrib/MacPDF/compile_xib_resources.gni")
+
+assert(current_os == "mac", "This application is macOS specific")
+
+group("MacPDF") {
+  deps = [ ":MacPDF.app" ]
+}
+
+compile_xib_resources("pdf_nibs") {
+  sources = [ "MainMenu.xib" ]
+}
+
+executable("MacPDF_executable") {
+  include_dirs = [
+    "//Userland/Libraries",
+    "//Userland/Services",
+  ]
+  defines = [ "AK_DONT_REPLACE_STD" ]
+  cflags_objcc = [ "-fobjc-arc" ]
+  deps = [
+    ":pdf_nibs",
+    "//AK",
+    "//Userland/Libraries/LibCore",
+    "//Userland/Libraries/LibGfx",
+    "//Userland/Libraries/LibPDF",
+  ]
+  frameworks = [
+    "Cocoa.framework",
+    "UniformTypeIdentifiers.framework",
+  ]
+  sources = [
+    "AppDelegate.mm",
+    "MacPDFDocument.mm",
+    "MacPDFOutlineViewDataSource.mm",
+    "MacPDFView.mm",
+    "MacPDFWindowController.mm",
+    "main.mm",
+  ]
+  output_name = "MacPDF"
+}
+
+bundle_data("pdf_bundle_info_plist") {
+  sources = [ "Info.plist" ]
+  outputs = [ "{{bundle_contents_dir}}/Info.plist" ]
+}
+
+bundle_data("pdf_bundle_executables") {
+  public_deps = [ ":MacPDF_executable" ]
+  sources = [ "$root_out_dir/bin/MacPDF" ]
+  outputs = [ "{{bundle_executable_dir}}/{{source_file_part}}" ]
+}
+
+bundle_data("pdf_bundle_libs") {
+  public_deps = [
+    "//Userland/Libraries/LibCompress",
+    "//Userland/Libraries/LibCore",
+    "//Userland/Libraries/LibCrypto",
+    "//Userland/Libraries/LibFileSystem",
+    "//Userland/Libraries/LibGfx",
+    "//Userland/Libraries/LibIPC",
+    "//Userland/Libraries/LibPDF",
+    "//Userland/Libraries/LibTextCodec",
+  ]
+  sources = [
+    "$root_out_dir/lib/liblagom-compress.dylib",
```

Comment:
Does this work? I'd imagine that the exe has an rpath of @excecutable_path/../lib to get from bin/ to lib/…oh, that's the right rpath to get from Contents/ to ../lib/ in the bundle too. Sneaky.

Reply:
I copied this from your own example you gave me :P https://github.com/nico/llvm-project/commit/314b632c3359c4031cc22819a551b0c1528c58ec#diff-308089ebf32771331ba28d58dcdda842a15968b7361921c9bf5d810493d8f2a4R9-R13

Reply:
Very clever, past me!